tools/toolchain/scripts/stage8/install_pugixml.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -e
+
+# TODO: Review and if possible fix shellcheck errors.
+# shellcheck disable=all
+
+[ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")/.." && pwd -P)"
+
+pugixml_ver="1.14.0"
```

Comment:
Please change to `1.14` to match pugixml's official release version: https://github.com/zeux/pugixml/releases/tag/v1.14